.travis.yml

```diff
@@ -0,0 +1,67 @@
+language: php
+
+notifications:
+  email: false
+
+sudo: required
+
+matrix:
+  fast_finish: true
+  include:
+     - php: "7.0"
```

Comment:
7.1 already maybe?

Reply:
…I also had the same question — but maybe later all `travis` and `scrutinizer` configs should be re-checked for all `"spryker-eco"` modules (to make uniform and consistent)…

Reply:
We officially dropped support for <=7.0 since 1st of November.